test/studies/IMSuite/vertexColor/vertexColoring-serial.chpl

```diff
@@ -0,0 +1,319 @@
+ /* This file is part of IMSuite Benchamark Suite.
+
+    This file is licensed to You under the Eclipse Public License (EPL);
+    You may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.opensource.org/licenses/eclipse-1.0.php
+
+   (C) Copyright IMSuite 2013-present.
+ */
```

Comment:
@benharsh: How about making the first line into "This file was derived from the IMSuite Benchmark Suite."  Or maybe even better, merging some of the comment just below that assert's Kushal's authorship into that first comment so that it starts with what the code is, lists the original and porting authors, and then states that it's a port from the "IMSuite benchmark suite which was released under the following license."  That makes the context clearer while still adhering the the license I believe.  (I.e., I don't think there's any requirement that the license information be the first thing in any file that's a derivative work, and I think it would be clearer if it wasn't).